package.json

```diff
@@ -20,7 +20,7 @@
     "start": "npm run build && npx probot run ./dist/bin/main.js",
```

Comment:
Rollback to the original start script, would look into why the probot command breaks on my machine which is node v14 and Windows 10.